system/jlib/jsocket.hpp

```diff
@@ -246,14 +246,14 @@ class jlib_decl ISocket : extends IInterface
 
     // Create client socket connected to a multicast server socket
     //
-    static ISocket*  multicast_connect( unsigned short port, const char *mcgroupip, unsigned ttl);
-    static ISocket*  multicast_connect( const SocketEndpoint &ep, unsigned ttl);
+    static ISocket*  multicast_connect( unsigned short port, const char *mcgroupip, unsigned _ttl=0);
```

Comment:
If we remove jbroadcast and thus all other places where multicast_connect/create are called then yes I think we should remove the default value.  I will do this.